lib/js_of_ocaml/json.ml

```diff
@@ -20,6 +20,88 @@
 open Js
 open! Import
 
+(****)
+
+let write_string buffer s =
+  Buffer.add_char buffer '\"';
+  for i = 0 to String.length s - 1 do
+    match s.[i] with
+    | '\"' -> Buffer.add_string buffer "\\\""
```

Comment:
Maybe use quoted string on the RHS so get rid of the escaping.

Reply:
Done